arwen/hosttest/execution_test.go

```diff
@@ -2267,6 +2268,45 @@ func TestExecution_CreateNewContract_Fail(t *testing.T) {
 		})
 }
 
+func TestExecution_CreateNewContract_IsSmartContract(t *testing.T) {
+
+	childCode := test.GetTestSCCode("deployer-child", "../../")
+
+	ownerNonce := uint64(23)
+	parentAddress := testcommon.MakeTestSCAddress("newAddr_" + fmt.Sprint(24))
+	childAddress := testcommon.MakeTestSCAddress("newAddr_" + fmt.Sprint(25))
```

Comment:
I usually write something like this, how it is here is also fine
```suggestion
	childAddress := testcommon.MakeTestSCAddress(fmt.Sprintf("newAddr_%d", 25))
```